shared/metrics/src/main/java/io/pravega/shared/MetricsNames.java

```diff
@@ -25,6 +25,12 @@
     public static final String HDFS_READ_BYTES = "tier2_read_bytes";          // Counter
     public static final String HDFS_WRITE_BYTES = "tier2_write_bytes";        // Counter
 
+    //filesystem stats
+    public static final String FILESYSTEM_READ_LATENCY = "filesystem_read_latency_ms";   // Timer
+    public static final String FILESYSTEM_WRITE_LATENCY = "filesystem_write_latency_ms"; // Timer
+    public static final String FILESYSTEM_READ_BYTES = "filesystem_read_bytes";          // Counter
+    public static final String FILESYSTEM_WRITE_BYTES = "filesystem_write_bytes";        // Counter
```

Comment:
In order to be compliant to existing and suggested naming https://github.com/pravega/pravega/issues/1370#issuecomment-307847464, change these to 
```
public static final String FILESYSTEM_READ_LATENCY = "tier2_read_latency_ms";   // Timer
public static final String FILESYSTEM_WRITE_LATENCY = "tier2_write_latency_ms"; // Timer
public static final String FILESYSTEM_READ_BYTES = "tier2_read_bytes";          // Counter
public static final String FILESYSTEM_WRITE_BYTES = "tier2_write_bytes";        // Counter
```
The prefix `filesystem` will be taken from createStatsLogger method as you have already added.